src/routes/authenticationRoute.js

```diff
@@ -112,4 +112,303 @@ authenticationRouter.post('/signup', validateUserSignup, signupAgent);
  */
 authenticationRouter.post ('/login', agentLogin);
 
-export default authenticationRouter;
\ No newline at end of file
+/**
+ * @swagger
+ * definitions:
+ *   Property:
+ *     properties:
+ *       image:
+ *         type: string
+ *       title:
+ *         type: string
+ *       address:
+ *         type: string
+ *       city:
+ *         type: string
+ *       landArea:
+ *         type: string
+ *       noOfBath:
+ *         type: string
+ *       noOfBed:
+ *         type: string
+ *       landSize:
+ *         type: string
+ *     example: {
+ *       "image": game.png,
+ *       "title": "duplex",
+ *       "address": "isolo street",
+ *       "city": "Lagos state",
+ *       "landArea": "5",
+ *       "noOfBath": "3",
+ *       "noOfBed": "2",
+ *       "landSize": 200
+ *     }
+ */
+
+/**
+ * @swagger
+ * /agent/properties:
+ *   post:
+ *     tags:
+ *       - Property Details
+ *     description: Add a new property to the database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: properties
+ *         description: property object
+ *         in: body
+ *         required: true
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *       - name: access_token
+ *         in: header
+ *         description: an authentication header
+ *         required: true
+ *         type: string
+ *     responses:
+ *       201:
+ *         description: property uploaded successfully
+ *       400:
+ *         description: Bad input supplied
+ *       500:
+ *         description: Internal server error
+ */
+ authenticationRouter.post(
+   '/agent/properties', 
+   verifyToken, 
+   validatePropertyInput, 
+   createProperty
+   );   
+
+
+
+/**
+ * @swagger
+ * definitions:
+ *   Properties:
+ *     properties:
+ *       image:
+ *         type: string
+ *       title:
+ *         type: string
+ *       address:
+ *          type: string
+ *       city:
+ *         type: string
+ *       landArea:
+ *         type: string
+ *       noOfBath:
+ *         type: string
+ *       noOfBed:
+ *         type: string
+ *       landSize:
+ *         type: string
+ *     example: {
+ *       "image": game.png,
+ *       "title": "duplex",
+ *       "address": "20, Allen str,surulere",
+ *       "city": "Lagos state",
+ *       "landArea": 500,
+ *       "noOfBath": "3",
+ *       "noOfBed": "2",
+ *       "landSize": 200
+ *      }
+ */
+
+
+/**
+ * @swagger
+ * /properties:
+ *   get:
+ *     tags:
+ *       - Property Details
+ *     description: retrive all properties from database
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *         schema:
+ *           $ref: '#/definitions/Properties'
+ *     responses:
+ *       200:
+ *         description: property uploaded successfully
+ *     example: {
```

Comment:
Is the example not supposed to be an array of objects?

Reply:
Array have being added too all examples